Bottom: 3255655c8a0237d097e33aab5275e52eb288b30b
Top:    ce671c37b1f84ce52eba78a5cbad4c962db83106
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-05 20:39:23 +0000

Refresh of toy-rendering-attempt

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index c5ccc21..a9599d8 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1128,9 +1128,9 @@ ghid_init_renderer (int *argc, char ***argv, GHidPort *port)
   ghid_graphics_class.fill_pcb_polygon = ghid_fill_pcb_polygon;
   ghid_graphics_class.thindraw_pcb_polygon = ghid_thindraw_pcb_polygon;
 
-//  step_model_to_shape_master ("/home/pcjc2/gedasrc/pcb/git/src/cheyney_step/Resistor_vr68.step");
-  step_model_to_shape_master ("/home/pcjc2/gedasrc/pcb/git/src/cheyney_step/Ceramite_2500z_10kV.step");
-//  step_model_to_shape_master ("/home/pcjc2/gedasrc/pcb/git/src/cheyney_step/Filament_Transformer.step");
+//  step_model_to_shape_master ("/home/pcjc2/gedasrc/pcb/git/src/example_step/Resistor_vr68.step");
+  step_model_to_shape_master ("/home/pcjc2/gedasrc/pcb/git/src/example_step/Ceramite_2500z_10kV.step");
+//  step_model_to_shape_master ("/home/pcjc2/gedasrc/pcb/git/src/example_step/Filament_Transformer.step");
 }
 
 void
